portal-frontend/src/app/features/public/search/notice-of-intent-search-table/notice-of-intent-search-table.component.ts

```diff
@@ -34,7 +34,11 @@ export class NoticeOfIntentSearchTableComponent implements OnDestroy {
   @Input() statuses: ApplicationStatusDto[] = [];
   @Output() tableChange = new EventEmitter<TableChange>();
 
-  displayedColumns = ['fileId', 'ownerName', 'type', 'portalStatus', 'lastUpdate', 'government'];
+  displayedColumns = ['fileId', 'ownerName', 'type', 'portalStatus', 'outcome', 'lastUpdate', 'government'];
+  outcomeMapping: {[code:string] : string } = {
```

Comment:
Rather than having 3 constants, can you move this to the public search dto and use a single constants?
I also think the type: Record<string,string> is easier to understand than {[code:string] : string }.

Reply:
we dont have that in srw

Reply:
The goal is to DRY up the code, since its repeated 3 times, I think we should have it once even if it doesn't involved SRWs.